.github/workflows/intergration-test.yml

```diff
@@ -0,0 +1,46 @@
+name: "Integration Test"
+
+on: [push, pull_request]
+
+jobs:
+  integration_test:
+    name: "Integration Test"
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        go: [ "1.15", "1.16" ]
+        os: [ubuntu-latest]
+
+    steps:
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go }}
+
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Checkout python env
+        uses: actions/setup-python@v2
+        with:
+            python-version: '3.8'
+
+      - name: Checkout java env
+        uses: actions/setup-java@v1
+        with:
+            java-version: '11'
+
+      - name: Setup-hdfs env
+        uses: beyondstorage/setup-hdfs@master
+        with:
+            hdfs-version: '3.3.1'
```

Comment:
We can test on more supported versions like: 

- "2.10.1"
- "2.2.2"
- "3.3.0"
- "3.3.1"

Reply:
I try to add more versions, just like:
`hdfs: [ "2.10.1", "2.2.2", "3.3.0", "3.3.1"]`,but look like failure :)

Reply:
How about sending a PR to `setup-hdfs` to fix it?